layers/+emacs/org/funcs.el

```diff
@@ -67,3 +67,8 @@
 (defun spacemacs/org-trello-push-card ()
   (interactive)
   (org-trello-sync-card))
+
+(defun spacemacs/org-clock-jump-to-current-clock ()
+  (interactive)
+  (load "org-clock")
```

Comment:
In cases where you encounter issues with not yet autoloaded functions the proper approach is to only add the missing autoload rather than to load the entire file.

This should look like this:
```suggestion
   (autoload #'org-clock-jump-to-current-clock "org-clock")
```

Reply:
pr updated, thanks for the suggestion!